src/constants.js

```diff
@@ -331,6 +331,9 @@ export const X_REP_HINT_PNG_DIMENSIONS_DEFAULT: '1024x1024' = '1024x1024';
 export const X_REP_HINT_HEADER_DIMENSIONS_DEFAULT = `[jpg?dimensions=${X_REP_HINT_JPG_DIMENSIONS_DEFAULT}&paged=false,png?dimensions=${X_REP_HINT_PNG_DIMENSIONS_DEFAULT}]`;
 
 /* ------------------ Sidebar View ---------------------- */
+export const SIDEBAR_FORCE_KEY: 'bcs.force' = 'bcs.force';
```

Comment:
This cache key is used to control the sidebar expand/collapse state and isn't meant for other purposes. Please create a new key for this new use case.

Reply:
The purpose of this PR is to control the sidebar expand/collapse state based on a user interaction (e.g, what clicking the sidebar buttons is currently doing)

Reply:
Ah, I see. I thought we were trying to store the open/close state of the _modal_ in local storage. My mistake! That said, I'd still like to avoid exposing the state management outside of the sidebar components. Can we use `history.replace({ state: { open: true } });` in one of the task components, instead?